keyboards/helix/helix.c

```diff
@@ -1 +1,8 @@
 #include "helix.h"
+#include "ssd1306.h" 
+
+#ifdef SSD1306OLED
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+	return process_record_gfx(keycode,record) && process_record_user(keycode, record);
+}
+#endif
```

Comment:
On the Helix keyboard, it looks like customary for helix.c to be empty and put the code in rev1/rev1.c, rev2/rev2.c and pico/pico.c.

I think it is better to ask @MakotoKurauchi.

Reply:
And, I think it's better to move `#include "ssd1306.h"` in the `#ifdef` block.
Because other codes do so.

```c
keyboards/helix/pico/keymaps/default/keymap.c-#ifdef AUDIO_ENABLE
keyboards/helix/pico/keymaps/default/keymap.c-  #include "audio.h"
keyboards/helix/pico/keymaps/default/keymap.c-#endif
keyboards/helix/pico/keymaps/default/keymap.c-#ifdef SSD1306OLED
keyboards/helix/pico/keymaps/default/keymap.c:  #include "ssd1306.h"
keyboards/helix/pico/keymaps/default/keymap.c-#endif

keyboards/helix/ssd1306.c-
keyboards/helix/ssd1306.c-#ifdef SSD1306OLED
keyboards/helix/ssd1306.c-
keyboards/helix/ssd1306.c:#include "ssd1306.h"
keyboards/helix/ssd1306.c-#include "i2c.h"
keyboards/helix/ssd1306.c-#include <string.h>
```

Reply:
I moved `#include "ssd1306.h"` into the `#ifdef` block.
I will wait for an answer from @MakotoKurauchi as to whether to move the code from helix.c to rev1.c/rev2.c/pico.c .

Reply:
I tested it. I want the code to move from helix.c to rev1.c / rev2.c / pico.c.
Thank you.

Reply:
All right. I will move these codes.